91-grub-mkconfig.install

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# this file is installed by sys-kernel/installkernel-gentoo
+
+set -euo pipefail
```

Comment:
In general, `-e` sucks because it means that the program will terminate silently without any message indicating where it failed. You could technically `trap ... EXIT` and try to grab a backtrace but I think your `die`s will suffice.

Reply:
I removed that section altogether